src/theme.js

```diff
@@ -4,6 +4,7 @@ import '@fontsource-variable/inter'
 
 // color mode config
 const config = {
+  // TODO: Change default color mode to system?
```

Comment:
Can you still look into this TODO, @VeckoTheGecko?

Reply:
Done